contributing/tests.md

```diff
@@ -7,268 +7,52 @@ describes them and gives pointers on how to run/debug them.
 
 The test types from most preferred to least preferred are:
 
-1. Unit Tests
-1. Vscode-integration Tests
-1. Integration Tests
-1. System Tests
+1. Unit Tests - jest: Found under the test/jest directory.
```

Comment:
On line 5 it says there are "several" kinds of tests, and after these updates there are actually only 2 that are described in this doc.  Maybe it could say some thing like, there are several, but moving forward we are focused on these two test types...

Reply:
nevermind - I see that the others are listed as deprecated so I think its fine as is